src/GraphQL.ApiTests/net6/GraphQL.approved.txt

```diff
@@ -3022,6 +3022,7 @@ namespace GraphQL.Validation
     public readonly struct ValidationOptions
     {
         public ValidationOptions() { }
+        public ValidationOptions(GraphQL.Types.ISchema schema, GraphQLParser.AST.GraphQLDocument document, System.Collections.Generic.IEnumerable<GraphQL.Validation.IValidationRule>? rules, System.Collections.Generic.IDictionary<string, object?> userContext, GraphQL.Instrumentation.Metrics metrics, GraphQL.Inputs variables, GraphQL.Inputs extensions, GraphQLParser.AST.GraphQLOperationDefinition operation, System.IServiceProvider? requestServices, System.Security.Claims.ClaimsPrincipal? user, System.Threading.CancellationToken cancellationToken) { }
```

Comment:
This is a read-only struct.  To allow C# 8 and prior languages to write to these properties, a constructor was added as a non-breaking change.  Changing the struct to a read/write struct is a breaking change and affects other code.